edweiss-app/components/lectures/slides/StudentQuestion.tsx

```diff
@@ -82,11 +82,14 @@ const StudentQuestion: ReactComponent<{ courseName: string, lectureId: string, q
 
         return (
             <>
-                <TView key={index} mb={'sm'} backgroundColor={isUser ? 'overlay0' : 'crust'} borderColor='surface0' radius={'lg'} flex={1} flexDirection='column' ml='md' style={{ right: isUser ? 0 : 10 }}>
-                    <TView borderColor={isUser ? 'crust' : 'overlay0'} bb={1} flexDirection='row' flexColumnGap={10} alignItems='center' radius={'lg'} mb={'sm'}>
-                        <Avatar size={40} name={anonym ? undefined : username} uid={anonym ? undefined : userID} />
-                        <TView flex={1}>
-                            <TText ml={16} mb={4} size={'sm'} pl={2} pt={'sm'}>{anonym ? "Anonymous" : username}, {Time.agoTimestamp(postedTime)}</TText>
+                {!answered && <TView key={index} mb={'sm'} backgroundColor='crust' borderColor='surface0' radius={'lg'} flex={1} flexDirection='column' mr={isUser ? 'md' : 'lg'} ml={isUser ? 'lg' : 'md'} style={{ right: isUser ? 0 : 10 }}>
+                    <TView pt={'sm'} flexDirection='row' flexColumnGap={10} alignItems='center' radius={'lg'} mb={'sm'}>
+                        <TView ml={'md'}>
+                            <Avatar size={30} name={anonym ? undefined : username} uid={anonym ? undefined : userID} />
+                        </TView>
+                        <TView flex={1} flexDirection='row' alignItems='flex-end' justifyContent='space-between'>
+                            <TText mb={4} size={'sm'} pl={2} pt={'sm'}>{anonym ? "Anonymous" : username}</TText>
```

Comment:
Maybe a minor detail.
Instead of putting in the `mb` a magic number 😁, you could use instead `mb='xs'`, as it fits the same value (4, in our case).
As you pointed it out in my previous PR, it's keeps think consitent  😉.

Reply:
Yes thank you I didn't notice it